util/timer/createTimer.js

```
@@ -1,16 +1,25 @@
+		if(timeInSeconds < 0){
+			return "The date must not be in the past."
+		}
+	}else {
+		timeInSeconds = parseTime(args[0], currentUnixTime);
+	} 
 	if (isNaN(timeInSeconds)) {
-		return message.channel.send("Please specify a time, and a message to send after the timer has finished")
+		return "Please specify a time, and a message to send after the timer has finished"
 	}
-	let customMessage = compatibility ? args.slice(1).join(" ") : args.slice(2).join(" ");
+	let customMessage = args.slice(1).join(" ")
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'customMessage' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)